TestApp/app/screens/CrashesScreen.js

```diff
@@ -131,6 +147,14 @@ export default class CrashesScreen extends Component {
                   title: 'Crash native code',
                   action: this.nativeCrash
                 },
+                {
+                  title: 'Generate low memory warning',
```

Comment:
This one only crashes and is not captured (either crash or warning).
Other platforms have only 1 button, that works.
Let's keep only the native one.

Reply:
Have you tested this on standalone release builds (ios & android)?
It works reliably for me.

Reply:
I was in debug. Release works indeed.